CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Main (unreleased)
 ### Breaking changes
 
 - Set `retry_on_http_429` to `true` by default in the `queue_config` block in static mode's `remote_write`. (@wildum)
+- windows_exporter adfs collector update.
```

Comment:
Is this a breaking change really? 

If so, once we're done upgrading our dependency properly, we should add some details around _why_ is this a breaking change. Any new or removed config fields should be accounted for in documentation and should also be reflected in the two `release-notes.md` documents

Reply:
apologies, i put that in originally thinking it was, but it turns out it was just bringing the doc's inline with the underlying code. https://github.com/prometheus-community/windows_exporter/pull/1272

I've updated the CHANGELOG file